### ISSUE TITLE
Navigation isn't working 

### ISSUE THREAD

**Issue by dirajkumar** [seanWLawrence/sveltekit-aws-lambda-demo#1]
I followed your blog and I was able deploy the application successfully. However, I couldn't get the navigation working. 

In my app i have "/app" , "/login" routes 

Any thoughts?

**Comment by seanWLawrence** [comment]
Yes, there are a few things I've been running into as I've started using more features of Sveltekit too, like handling forms, that I'll be writing a follow up post on. 

For now I'll also update the existing post to add the known issues. Thank you for reminding me. 

For the path name error you're having, set the ORIGIN environment variable in the lambda function to the deployed URL

**Comment by seanWLawrence** [comment]
@dirajkumar I just got the other known issue working - POST requests. You can see the source code in the dev branch infra folder [here](https://github.com/seanWLawrence/cobra-events/tree/dev/infra) as an example. The cloudfront, server and deploy files are the most relevant
